docs/customize.md

```diff
@@ -167,7 +167,7 @@ Disables gutter icons when doing multiple selections.
 ```
 
 ## Tag Plugin Settings
-Tag settings found in `bh_core.sublime-settings`.
+Tag settings found in `bh_tag.sublime-settings`.
```

Comment:
Coolio, that's what I figured. I added a context menu item in PR #271 so people can access that file from the Sublime preferences menu.